src/eduid_userdb/proofing/proofingdb.py

```diff
@@ -250,6 +286,55 @@ def remove_state(self, state):
         self.remove_document({'eduPersonPrincipalName': state.eppn,
                         'verification.email': state.verification.email})
 
+    def remove_state_phone(self, state):
```

Comment:
I think this should be moved to PhoneProofingStateDB and just called remove_state.